### PR TITLE
Fixed names to have consistent case

### PR DESCRIPTION
**Pull request by nadenf**

### CODE REVIEW

File: kubeinit/roles/kubeinit_eks/tasks/20_configure_common.yml [hunk 1]
```diff
@@ -268,7 +268,7 @@
           - "kubectl-{{ kubeinit_eks_kubernetes_version }}"
         state: present
 
-    - name: enable kubelet
+    - Enable kubelet
```
Comment (ccamacho) [inline kubeinit/roles/kubeinit_eks/tasks/20_configure_common.yml:271]: missing -name:

File: kubeinit/roles/kubeinit_libvirt/tasks/10_cleanup.yml [hunk 2]
```diff
@@ -25,6 +25,7 @@
     - name: Get all the networks
       community.libvirt.virt_net:
         command: list_nets
+        name: ""
```
Comment (ccamacho) [inline kubeinit/roles/kubeinit_libvirt/tasks/10_cleanup.yml:28]: Remove

File: kubeinit/roles/kubeinit_okd/tasks/10_configure_service_nodes.yml [hunk 3]
```diff
@@ -278,31 +278,31 @@
...
         openshift-install create manifests --dir=install_dir/
       register: render_bootstrap_details
       changed_when: "render_bootstrap_details.rc == 0"
 
-    - name: "Enable master schedulable if there are no worker nodes"
+    - name: Enable master schedulable if there are no worker nodes"
       ansible.builtin.shell: |
         set -o pipefail
         cd
-        yaml_file="install_dir/manifests/cluster-scheduler-02-config.yml"
```
Comment (ccamacho) [inline kubeinit/roles/kubeinit_okd/tasks/10_configure_service_nodes.yml:297]: do not remove the final quotes

File: kubeinit/roles/kubeinit_prepare/tasks/prepare_groups.yml [hunk 4]
```diff
@@ -18,94 +18,94 @@
...
 
-- name: add localhost to the local_host group
+- name: Add localhost to the local_host group
   ansible.builtin.add_host:
     name: kubeinit-localhost
     group: local_host
     ansible_host: localhost
-    ansible_hostname: localhost
-    inventory_hostname: localhost
+    ansible_hostname: Localhost
```
Comment (ccamacho) [inline kubeinit/roles/kubeinit_prepare/tasks/prepare_groups.yml:75]: Do not use L

**Review by ccamacho — COMMENTED** [review]
Thank you for your contributions!!! This is awesome!
Just a few minor changes, also you need to adjust the commit title (Not the pull request title) to be something like:
`fix: adjust names to have consistent case`

This is because we lint the commit titles for the changelog, its explained here: http://docs.kubeinit.com/contributing.html#linting-commits

In any case, again, thanks for your contributions!

File: kubeinit/roles/kubeinit_prepare/tasks/40_ovn_setup.yml [hunk 5]
```diff
@@ -21,7 +21,7 @@
 - name: Set up the OVN network details
   block:
 
-    - name: configure OVS on the Hypervisors
+    - Configure OVS on the Hypervisors
```
Comment (ccamacho) [inline kubeinit/roles/kubeinit_prepare/tasks/40_ovn_setup.yml:24]: missing name:

Comment (ccamacho) [reply kubeinit/roles/kubeinit_prepare/tasks/40_ovn_setup.yml:24]: Also, this branch is not up to date with main, would you mind to rebase it on top of main?

**Comment by nadenf** [comment]
One of the messiest PRs in a long time but we got there in the end !

**Pull request merged**